src/middlewares/__tests__/createInsightsMiddleware.ts

```diff
@@ -350,7 +349,7 @@ describe('insights', () => {
           insightsClient,
         })({ instantSearchInstance });
         middleware.subscribe();
-        expect(getUserToken()).toEqual(ANONYMOUS_TOKEN);
+        expect(getUserToken()).toEqual('token-from-queue-before-init');
       });
```

Comment:
This is technically a breaking change, but the previous behavior was wrong. If user calls `init` twice, the second `init` shouldn't override the previous usertoken with anonymous usertoken (ref: https://github.com/algolia/search-insights.js/pull/251)